src/main/java/uk/gov/beis/els/categories/spaceheaters/model/SpaceHeaterPackagesCalculatorForm.java

```diff
@@ -0,0 +1,191 @@
+package uk.gov.beis.els.categories.spaceheaters.model;
+
+import javax.validation.constraints.Digits;
+import javax.validation.constraints.NotNull;
+import org.hibernate.validator.group.GroupSequenceProvider;
+import uk.gov.beis.els.categories.common.StandardTemplateForm50Char;
+import uk.gov.beis.els.model.meta.FieldPrompt;
+
+@GroupSequenceProvider(SpaceHeaterPackagesCalculatorFormSequenceProvider.class)
+public abstract class SpaceHeaterPackagesCalculatorForm extends StandardTemplateForm50Char {
+
+  @FieldPrompt("Rated heat output (Prated) of the preferential heater (in kW)")
+  @Digits(integer = 4, fraction = 0, message = "Enter the heat output of the preferential heater")
+  private String preferentialHeaterHeatOutput;
+
+  @FieldPrompt("Seasonal space heating energy efficiency of the preferential heater (in %)")
+  @Digits(integer = 3, fraction = 0, message = "Enter the space heating energy efficiency of the preferential heater")
+  private String preferentialHeaterSeasonalSpaceHeatingEfficiencyPercentage;
+
+  @FieldPrompt("Temperature control installed?")
+  @NotNull(message = "Specify if this package includes temperature control")
+  private Boolean hasTemperatureControl;
+
+  @FieldPrompt("Temperature control class")
+  @NotNull(message = "Select the temperature control class",
+      groups = TemperatureControlGroup.class)
+  private String temperatureControlClass;
+
+  @FieldPrompt("Supplementary boiler installed?")
+  @NotNull(message = "Specify if this package includes a supplementary boiler")
+  private Boolean hasSupplementaryBoiler;
+
+  @FieldPrompt("Rated heat output (Prated) of the supplementary boiler (in kW)")
+  @Digits(integer = 4, fraction = 0, message = "Enter the rated heat output of the supplementary boiler",
+      groups = SupplementaryBoilerGroup.class)
+  private String supplementaryBoilerHeatOutput;
+
+  @FieldPrompt("Seasonal space heating energy efficiency of supplementary boiler (in %)")
+  @Digits(integer = 3, fraction = 0, message = "Enter the seasonal space heating energy efficiency of supplementary boiler",
+      groups = SupplementaryBoilerGroup.class)
+  private String supplementaryBoilerSeasonalSpaceHeatingEfficiencyPercentage;
+
+  @FieldPrompt("Solar collector installed?")
+  @NotNull(message = "Specify if this package includes a solar collector")
+  private Boolean hasSolarCollector;
+
+  @FieldPrompt("Solar collector area (in m2)")
+  @Digits(integer = 3, fraction = 0, message = "Enter the solar collector area",
+      groups = SolarCollectorGroup.class)
+  private String solarCollectorSize;
+
+  @FieldPrompt("Solar collector efficiency (in %)")
+  @Digits(integer = 3, fraction = 0, message = "Enter the solar collector efficiency",
+      groups = SolarCollectorGroup.class)
+  private String solarCollectorEfficiencyPercentage;
+
+  @FieldPrompt("Heat storage tank installed?")
```

Comment:
@matteason is 'Heat storage tank' here correct or should it be a 'Hot water storage tank'?
The EU tool seems to use both interchangeably, but the energy label uses 'Hot water storage tank'. And given it ask for the energy rating of the tank from the label in a nested question it should presumably be referred to as a 'Hot water storage tank'? 
![image](https://user-images.githubusercontent.com/3612419/158167914-b2dfdb67-02df-4d6a-bf8e-d012ad303b0c.png)

Reply:
changed to "hot water storage tank"
happy to revert change if necessary